src/app/app.component.html

```diff
@@ -36,6 +36,14 @@
               </ion-label>
             </ion-item>
           </ion-menu-toggle>
+          <ion-menu-toggle auto-hide="false">
+            <ion-item routerDirection='root' routerLink="/files">
+              <ion-icon slot="start" name="document"></ion-icon>
+              <ion-label>
+                Documents
```

Comment:
not sure Documents is the best name on the side bar. It doesn't tell me straight away what this tab does. Maybe File Upload, Upload, File Storage or something similar?

Reply:
I have tried to organize that around business needs. FileUpload is just one of the features there.
We have file metadata storage,  FileQuery and also FileDownload feature. 
So maybe Files? But then files can be images and they require some specific cordova plugins.
We provide file upload but not Camera plugin side of functionality so documents seemed better initially. With that in mind what will be your pick?

Reply:
Yeah OK maybe you got it right first time around then. Ignore me.